{"name": "test_login_noExistUser_wrongPasswd[data1]", "status": "passed", "attachments": [{"name": "log", "source": "f168da23-18ce-4f53-8627-efac77403c47-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'用户名': 'dsadsadsa', '密码': 'pytho', '预期': '请输入至少6位数的密码'}"}], "start": 1587375141415, "stop": 1587375142274, "uuid": "0b605eb0-8f89-41e9-b91c-cecc9a16732a", "historyId": "634fa2f876868d6c8f81ec03e35c966f", "testCaseId": "d35b8259a99be3473adfcaff1df8682d", "fullName": "TestCases.test_00_login.TestLogin#test_login_noExistUser_wrongPasswd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "17492-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_00_login"}]}